modules/balana-utils/pom.xml

```diff
@@ -32,19 +32,23 @@
     </dependencies>
 
     <build>
+        <pluginManagement>
```

Comment:
Can you please explain the reason to move incremental-build-plugin to pluginManagement section?

Thanks,